metaci/plan/forms.py

```diff
@@ -86,8 +91,20 @@ def _get_branch_choices(self):
                 choices.append((branch.name, branch.name))
         return tuple(choices)
 
-    def create_build(self):
+    def clean(self):
+        if is_release_plan(self.plan):
+            release = self.cleaned_data.get("release")
+            if not release:
+                self.add_error("release", "Please specify the release.")
```

Comment:
Clarifying: the requirement to link a release plan to a release will apply to all MetaCI consumers, but the change case requirement is gated on the environment variable. Right?

Reply:
@davidmreed Yeah, good distinction. That is correct.